bali/core.py

```diff
@@ -26,6 +26,9 @@ class Settings(BaseSettings):
                 'ROUTING_KEY': 'test_routing_key'
             }
     }
+    EVENT_TO_AMQP_MAP: Dict[str, str] = {
```

Comment:
If there's only one AMQP and only one event stream, the map should not configurable.

Configurable makes it complex.